src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -126,11 +127,11 @@ bool AdaptDispatch::_CursorMovement(const CursorDirection dir, const size_t dist
             case CursorDirection::Down:
             case CursorDirection::NextLine:
             case CursorDirection::PrevLine:
-                pModify = &cursor.Y;
+                modify = cursor.Y;
```

Comment:
**DANGER, WILL ROBINSON**

References cannot be re-seated! This is a write-through into `cursor.X`

Reply:
dang. ok. I thought I was being creative. I'll re-pointer it.